EMS/core-bundle/src/Service/Form/Submission/FormSubmissionService.php

```diff
@@ -236,9 +236,13 @@ public function submit(DatabaseRequest $submitRequest): array
         ];
     }
 
-    public function removeExpiredSubmissions(): int
+    public function removeExpiredSubmissions(bool $keepMetadata): int
     {
-        return $this->formSubmissionRepository->removeAllOutdatedSubmission();
```

Comment:
Test if attachments are also deleted (CASCADE DELETE) or if they need to be deleted before